client/my-sites/stats/stats-reach/index.jsx

```diff
@@ -35,29 +35,29 @@ export const StatsReach = props => {
 		<div>
 			{ siteId && <QuerySiteStats siteId={ siteId } statType="statsFollowers" /> }
 			{ siteId && <QuerySiteStats siteId={ siteId } statType="statsPublicize" /> }
-			<SectionHeader label={ translate( 'Followers' ) } />
+			<SectionHeader label={ translate( 'Follower Totals' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **12** times:
`translate( ' Total Followers' )` *ES Score: 11.74*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/stats/insights-columns)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).